akka-docs/rst/scala/stream/stream-quickstart.rst

```diff
@@ -223,7 +223,7 @@ combinator:
   due to the risk of deadlock (with merge being the preferred strategy), and second, the monad laws would not hold for
   our implementation of flatMap (due to the liveness issues).
 
-  Please note that the ``mapConcat`` requires the supplied function to return a strict collection (``f:Out=>immutable.Seq[T]``),
+  Please note that the ``mapConcat`` requires the supplied function to emit an iterable (``f: Out => immutable.Iterable[T]``),
```

Comment:
Nice catch. Do you have a supporting idea behind return -> emit change? For me emit is about pushing values via a side-channel, like our `GraphStage` emits elements, or a tokenizer emits token in compilers. Conversely, here `Iterable` refers to the value being returned by the provided function.